lib/util/edit.test.ts

```diff
@@ -289,6 +289,65 @@ function a2() {}`,
   });
 
   describe('class declaration', () => {
+    it('should not remove decorators when exports are deleted', () => {
+      const fileService = new MemoryFileService();
+
+      fileService.set('/app/main.ts', ``);
+      fileService.set(
+        '/app/a.ts',
+        `@myDecorator
+export class A {}
+const a = new A();
+console.log(a);`,
```

Comment:
```suggestion
export class A {}
```
can you remove the class instance and console.log from the added test cases to keep it lean?

Reply:
ah yes you're right, done